README.md

```diff
@@ -45,10 +45,11 @@ To contribute, please check out our contribution guide [here](https://github.com
 
 ---
 
-##AngularJS
+## AngularJS
 
--[Angular Academy](https://www.angularacademy.ca/)
--[W3 Schools](https://www.w3schools.com/angular/)
+- [Angular Academy](https://www.angularacademy.ca/)
+
```

Comment:
Please remove this extra space between items

Reply:
resolved